temba/channels/types/facebook/type.py

```diff
@@ -51,10 +51,26 @@ def deactivate_trigger(self, trigger):
         if trigger.trigger_type == Trigger.TYPE_NEW_CONVERSATION:
             self._set_call_to_action(trigger.channel, None)
 
+    @classmethod
+    def format_quick_replies(cls, quick_replies):
+        data = json.loads(quick_replies)
+        data = data.get('quick_replies', None)
+        replies = []
+
+        if data:
+            replies = [dict(title=item, payload=item, content_type='text') for item in data]
+
+        return replies
+
     def send(self, channel, msg, text):
         # build our payload
         payload = {'message': {'text': text}}
 
+        metadata = msg.metadata if hasattr(msg, 'metadata') else None
```

Comment:
Seems like this could be simpler, e.g.

```python
metadata = json.loads(msg.metadata) if hasattr(msg, 'metadata') else {}
quick_replies = metadata.get('quick_replies', [])
formatted_replies = [dict(title=item, payload=item, content_type='text') for item in quick_replies]
```

But really we should put the metadata into redis as a dict, and pull it out as a dict, so individual channels don't have to parse the JSON each time